prime-router/settings/organizations-test.yml

```diff
@@ -450,12 +451,15 @@
       organizationName: nj-doh
       topic: covid-19
       jurisdictionalFilter:
-        - matches(ordering_facility_state, NJ)
+        - matches(ordering_facility_state, NJ, patient_state, NJ)
```

Comment:
Just noting that this is different from both prod and local -- no filters here on processing_mode_code.